teryt/utils_zip.py

```diff
@@ -0,0 +1,80 @@
+import os.path

```

Comment:
To będzie drobiazg, ale ułatwiający robienie zmian.

Jak robisz importy to jest taka dobra praktyka:
- Dzielimy je na trzy grupy. W pierwszej są importowane rzeczy wbudowane w Pythona, w drugiej rzeczy z zewnętrznych bibliotek, a w trzeciej lokalne importy.
- W każdej z grup dobrze jest ułożyć importy w kolejności alfabetycznej. Jak będzie miał pliki, gdzie jest ich bardzo dużo to zobaczysz jak to pomaga ;)